plugins/chainlink.Dockerfile

```diff
@@ -16,10 +16,20 @@ COPY . .
 
 # Build the golang binaries
 RUN make install-chainlink
-RUN make install-solana
+# Build LOOP Plugins
 RUN make install-median
 RUN make install-starknet
 
+RUN go list -m -f "{{.Dir}}" github.com/smartcontractkit/chainlink-solana | xargs -I % ln -s % /chainlink-solana
+
+# Build image: Solana Plugin
+FROM golang:1.21-bullseye as buildsol
+RUN go version
+WORKDIR /chainlink-solana
+
+COPY --from=buildgo /chainlink-solana .
+RUN go install ./pkg/solana/cmd/chainlink-solana
```

Comment:
nit - is this resusable somehow? seems like building starknet/other chains would probably use something similar

Reply:
In the short term, it can be reused as one build image for all the plugins.
It is just a work around for `go install` being incompatible with `replace`s in the `go.mod`. Longer term, we don't want to be installing from source anyways, so this complication goes away.